README.adoc

```diff
@@ -129,6 +129,17 @@ Below is a guide for setting up the Heroku application from scratch should there
 
 == Environment variables
 
+`PROXY`::
+(Required) Set to `true` if se-edu-bot is served behind a reverse proxy (e.g. ngrok or heroku).
+Given that we host se-edu-bot on heroku and use ngrok for development,
+this should usually be set to `true`.
```

Comment:
Actually, given that for both development and production this should be set to `true`, we shouldn't even need this environment variable in the first place?